data/augment.py

```diff
@@ -0,0 +1,768 @@
+"""
+
+This data recipe lets the user to augment new features to the dataset using the Augment Cloud Service
+1. The recipe pushes the dataset to user's snowflake account
+2. The recipe imports the dataset into augment service
+3. The recipe submits request for augmentation
+4. The recipe polls the API for the matching engine to process the request
+5. The recipe submits job for the creation of new augment table. The service uses correlation to find relevance between the new features and the given target label
+6. The recipe polls the API for the completion of the table creation
+6. The recipe exports the dataset back to user's snowflake account
+7. The recipe downloads, saves the dataset from snowflake into driverlessai instance and returns the file path
+8. A new dataset is created in DAI with the augmented columns
```

Comment:
customer facing recipe i would use full product name

Reply:
@surenH2oai  I have it updated now :)